main_view.html

```diff
@@ -8,44 +8,324 @@
+                            <li class="tools"><button value="Input Buttom" type="submit">css</button></li>
+                             <li class="tools"><button value="Input Buttom" type="submit">Javascript</button></li>
+                             <li class="tools"><button value="Input Buttom" type="submit">html</button></li>
+                </ul>
+                            
+                
+                <button class="see_project" type="submit">see project</button>
+                
+            </div>
+        </div>
+            </div>
+    </div>
+        
+        <div class="recent_works">
+            <div class="recent_works_elements">
+              
+                
+                
+                <div class="recent_works_element">
+                    <div class="works_img"> </div>
+                <div class="works_button_action">
+                    <p class="works_title">Multi-Post Stories Gain+Glory </p>
+                    
+                    <ul class="works_buttons">
+                        
+                        <li class="tools"><button value="Input Buttom" type="submit">Ruby on rails</button> </li>
+                            <li class="tools"><button value="Input Buttom" type="submit">css</button></li>
+                             <li class="tools"><button value="Input Buttom" type="submit">Javascript</button></li>
+                             <li class="tools"><button value="Input Buttom" type="submit">html</button></li>
+                    </ul>
+                                
+                    
+                    <button class="see_project" type="submit">see project</button>
+                    
+                </div>
+            </div>
+                </div>
+        </div>
+
+            <div class="recent_works">
+                <div class="recent_works_elements">
+                  
+                    
+                    
+                    <div class="recent_works_element">
+                        <div class="works_img"> </div>
+                    <div class="works_button_action">
+                        <p class="works_title">Multi-Post Stories Gain+Glory </p>
+                        
+                        <ul class="works_buttons">
+                            
+                            <li class="tools"><button value="Input Buttom" type="submit">Ruby on rails</button> </li>
+                                <li class="tools"><button value="Input Buttom" type="submit">css</button></li>
+                                 <li class="tools"><button value="Input Buttom" type="submit">Javascript</button></li>
+                                 <li class="tools"><button value="Input Buttom" type="submit">html</button></li>
+                        </ul>
+                                    
+                        
+                        <button class="see_project" type="submit">see project</button>
+                        
```

Comment:
![Screenshot from 2022-08-21 05-18-45](https://user-images.githubusercontent.com/93039359/185770245-3e093309-ad47-41d7-9386-232c1625c496.png)

- [ ] Please add these effects to all your button in the project to follow the Fimga template :blush: